### ISSUE TITLE
Transaction label change is propagated only the first time 

### ISSUE THREAD

**Issue by louneskmt** [sparrowwallet/sparrow#184]
Adding a label to a transaction automatically set this label for the corresponding receiving address and utxos, but only on first labelling. If you edit a transaction label, address and utxos labels won't change. Is this intended?

**Comment by craigraw** [comment]
Yes, this is intended. Sparrow never wants to automatically overwrite an already set (non-empty) label in order to preserve any user data present.

**Comment by louneskmt** [comment]
Thanks for the answer @craigraw. Maybe add an option to propagate this change (right-click "propagate label")? Definitely not high priority.

**Comment by 6102bitcoin** [comment]
Feature request: Propagate label button to push update to label.

**Comment by oPFGKk9gtuw8nuHkzrQn** [comment]
This would be very helpful. Is there any documentation about the labeling?
